kuma/accounts/views.py

```diff
@@ -0,0 +1,12 @@
+import logging
+
+from django.shortcuts import render
+
+from kuma.core.decorators import login_required
+
+log = logging.getLogger("kuma.accounts.views")
+
+
+@login_required
```

Comment:
I'd rather we don't do this. Once we go to Yari this will never be a thing. You can't have the server redirect based on cookies unless it's `/api/v1/*` endpoints. See what we did for [the subscription management page](https://github.com/mdn/kuma/blob/3371363ed90751dcb5c495111d34d2c61a98537f/kuma/payments/views.py#L33-L35).
What the main React page component should do is:

1) Wait for `/api/v1/whoami`
2) If no user in `userData`, display [something about not being logged in](https://github.com/mdn/kuma/blob/3371363ed90751dcb5c495111d34d2c61a98537f/kuma/javascript/src/payments/pages/management.jsx#L151-L158).
3) If logged in, load the `/api/v1/account` (or whatever we chose)

This is the pattern of the future.